Brand-Guide.md

```diff
@@ -96,11 +80,11 @@ A clear space also known as an exclusion zone is an area of blank space that sur
 
 1. **Applying Effects**: Avoid adding visual effects such as drop shadows, glows, or bevels to the logo. These effects can distract and compromise the logo's legibility.
 
-1. **Adding Elements**: Do not augment the logo with additional elements or graphics. Adding elements can weaken the overall design and detract from its intended impact.
+1. ~~**Adding Elements**: Do not augment the logo with additional elements or graphics. Adding elements can weaken the overall design and detract from its intended impact.~~
```

Comment:
I crossed this off for your consideration since the BIDS Connectivity project did this and theirs still looks okay.

https://pestillilab.github.io/bids-connectivity/

Reply:
Oh yes, thank you